src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -69,13 +71,13 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
             onMouseLeavePlot: action(()=>{ this.mouseInPlot = false;}),
             handleDownloadDataClick:()=>{
                 let firstLine = this.props.clinicalAttribute.displayName+'\tCount'
-                fileDownload(firstLine+'\n'+this.plot.downloadData(), fileName);
+                fileDownload(firstLine+'\n'+this.plot.downloadData(), this.fileName);
             },
             handleSVGClick:()=>{
```

Comment:
i'm surprised that this works.  i would think the click handlers need to be bound to instance (which we can now with auto-bind decorator).

Reply:
The download feature is not updated in this pull request and not used in anyway currently. We will send out another pull request to fix the issue.